modules/validations/outdated_tags.js

```diff
@@ -79,7 +146,7 @@ export function validationOutdatedTags() {
       }
     }
 
-    // add missing addTags..
+    // Add missing addTags from the detected preset
     let newTags = Object.assign({}, entity.tags);  // shallow copy
     if (preset.tags !== preset.addTags) {
       Object.keys(preset.addTags).forEach(k => {
```

Comment:
Each flag entry in NSI comes with a `flag:name` tag, but that’s mainly for internal reference. The `flag:name` key would logically be in the local language, whereas the value in NSI is specific to English, so it wouldn’t be appropriate to add the `flag:name` from NSI in non-English-speaking countries. (Apologies if the code below already handles that; I couldn’t tell.)

Reply:
> (Apologies if the code below already handles that; I couldn’t tell.)

It doesn't but it should - this is a good catch..